src/main/java/com/cburch/logisim/fpga/designrulecheck/Netlist.java

```diff
@@ -2892,36 +2890,36 @@ private void WarningForGatedClock(
       for (NetlistComponent comp : Components.get(i))
         AlreadyWarned |= WarnedComponents.contains(comp);
       if (!AlreadyWarned) {
-        if (Sources.get(i).getSource().GetComp().getFactory() instanceof SubcircuitFactory) {
+        if (Sources.get(i).getSource().getComp().getFactory() instanceof SubcircuitFactory) {
           Reporter.Report.AddSevereWarning(S.get("NetList_GatedClock"));
           Reporter.Report.AddWarningIncrement(S.get("NetList_TraceListBegin"));
-          simpleDRCContainer warn =
-              new simpleDRCContainer(
-                      MyCircuit,
+          SimpleDRCContainer warn =
+              new SimpleDRCContainer(
+                      myCircuit,
                       S.get("NetList_GatedClockSink"),
-                      simpleDRCContainer.LEVEL_NORMAL,
-                      simpleDRCContainer.MARK_INSTANCE | simpleDRCContainer.MARK_WIRE,
+                      SimpleDRCContainer.LEVEL_NORMAL,
+                      SimpleDRCContainer.MARK_INSTANCE | SimpleDRCContainer.MARK_WIRE,
                       true);
-          warn.AddMarkComponents(Wires.get(i));
+          warn.addMarkComponents(Wires.get(i));
           for (NetlistComponent comp : Components.get(i))
-            warn.AddMarkComponent(comp.GetComponent());
+            warn.addMarkComponent(comp.getComponent());
           Reporter.Report.AddWarning(warn);
           WarningTraceForGatedClock(
               Sources.get(i).getSource(),
               Sources.get(i).getIndex(),
               HierarchyNetlists,
-              CurrentHierarchyLevel);
+                  currentHierarchyLevel);
           Reporter.Report.AddWarningIncrement(S.get("NetList_TraceListEnd"));
         } else {
-          simpleDRCContainer warn =
-              new simpleDRCContainer(
-                      MyCircuit,
+          SimpleDRCContainer warn =
+              new SimpleDRCContainer(
+                      myCircuit,
                       Warning,
-                      simpleDRCContainer.LEVEL_SEVERE,
-                      simpleDRCContainer.MARK_INSTANCE | simpleDRCContainer.MARK_WIRE);
+                      SimpleDRCContainer.LEVEL_SEVERE,
+                      SimpleDRCContainer.MARK_INSTANCE | SimpleDRCContainer.MARK_WIRE);
           for (NetlistComponent comp : Components.get(i))
-            warn.AddMarkComponent(comp.GetComponent());
-          warn.AddMarkComponents(Wires.get(i));
+            warn.addMarkComponent(comp.getComponent());
+          warn.addMarkComponents(Wires.get(i));
           Reporter.Report.AddWarning(warn);
         }
         WarnedComponents.addAll(Components.get(i));
```

Comment:
Maybe we should make this file smaller (I know that it is my code), ass 2927 lines is a lot :-)

Reply:
Do agree, but not as part of this PR